lib/rack/tracker.rb

```diff
@@ -12,6 +12,7 @@
 require "rack/tracker/handler_delegator"
 require "rack/tracker/controller"
 require "rack/tracker/google_analytics/google_analytics"
+require "rack/tracker/google_adwords_conversion/google_adwords_conversion"
```

Comment:
:+1:  but i think we should do it in a separate PR, because it affects all handlers